documentation/PATh/htc_workloads/submitting_workloads/quickstart.md

```diff
@@ -0,0 +1,362 @@
+---
+path:
+    path: htc_workloads/submitting_workloads/quickstart.md
+---
+
+# Quickstart - Submit Example HTCondor Jobs
+
+
+## Job 1: A simple, nonparallel job
+
+First create a `short.sh` file to use as en executable for our sample job:
+
+
+	#!/bin/bash
+	# short.sh: a short discovery job
+        set -e
+	printf "Start time: "; /bin/date
+	printf "Job is running on node: "; /bin/hostname
+	printf "Job running as user: "; /usr/bin/id
+	printf "Job is running in directory: "; /bin/pwd
+	echo
+	echo "Working hard..."
+	sleep 20
+	echo "Science complete!"
+
+Now, make the script executable.
+
+	$ chmod +x short.sh
+
+
+### Run the job locally
+
+When setting up a new job submission, it's important to test your job outside
+of HTCondor before submitting into pool. 
```

Comment:
```suggestion
When possible, it is important to first test your job on your local resorces prior to submitting the job to HTCondor to run on the PATh Facility execution points.  This will help identify potential errors within the code prior to queuing many HTCondor jobs. 

For example, we can run this executable on our local terminal by typing: 
```